tests/fast-integration/skr-test/README.md

```diff
@@ -42,51 +70,33 @@ KCP_KUBECONFIG_API_URL=
 AL_SERVICE_KEY= #must be a cloud foundry service key with the info about `UAA (User Account and Authentication)`. Learn more about [Managing Service Keys in Cloud Foundry](https://docs.cloudfoundry.org/devguide/services/service-keys.html).
 ```
 
-Run the following command to set up the environment variables in your system:
+2. To set up the environment variables in your system, run:
 
 ```bash
 export $(xargs < .env)
 ```
-### Usage:
-
-The SKR test can be used in two modes:
-
-#### With provisioning
-In this case the tests executes the following steps:
-   1. Provision SKR cluster and register compass resources
-   2. OIDC Test
-   3. Commerce Mock Test
-   4. De-provision the SKR instance and clean the compass resources
 
-Run the test y calling the following target:
-```bash
-npm run test-skr
-#or
-make test-skr
-```
+3. Choose whether you want to run the test with or without provisioning.
+   - To run the test **with** provisioning, call the following target:
 
-#### Without Provisioning. 
-In this mode the test will additionally need following environment variables:
-- `SKIP_PROVISIONING` needs be set to `true` in that case
-- `INSTANCE_ID` the uuid of the provisioned SKR instance
-- `TEST_SUFFIX` is used to determine the compass scenario and the app assigned to it. Will be randomly generated if not set.
+    ```bash
+    npm run test-skr
+    #or
+    make test-skr
+    ```
+    - To run the SKR test **without** provisioning, use the following command:
 
->**IMPORTANT:** The `TEST_SUFFIX must be 4 characters long and is required for every commerce-mock test run. In case the compass scenario with the given suffix already exists, the test will try to reuse it.
+    ```bash
+    make test-skr SKIP_PROVISIONING=true
+    #or
+    npm run test-skr # when all env vars are exported
+    ```
 
-This mode consists of the following steps:
-1. Ensure SKR exists and register compass resources
-2. OIDC Test
-3. Commerce Mock Test
-4. Clean the compass resources
+### CommerceMock resources preparation
 
-Run the SKR test in that mode:
-```bash
-make test-skr SKIP_PROVISIONING=true
-#or
-npm run test-skr # when all env vars are exported
-```
+The SKR test additionally contains a standalone script, which can be used to register the CommerceMock resources. Run this command to run if you want to prepare the CommerceMock resources:
```

Comment:
I added the first sentence to the "Usage Modes" section.

```suggestion
If you want to prepare the Commerce Mock resources

1. Set all the environment variables as explained for the mode without provisioning.
2. Run the following command:
```

Reply:
done